Watchman.Discord/WatchmanBot.cs

```diff
@@ -47,13 +49,24 @@ public WatchmanBot(DiscordConfiguration configuration)
         public async Task Start()
         {
             MongoConfiguration.Initialize();
-            ServerInitializer.Initialize(this._client);
+            ServerInitializer.Initialize(_client, _container);
 
             await _client.LoginAsync(TokenType.Bot, this._configuration.Token);
             await _client.StartAsync();
+            _client.Ready += UnmuteUsers;
```

Comment:
ale zaraz, dlaczego tak? jak to ma działać?

Reply:
Chodzi o to, że serwis ma unmutować wszystkich, którzy nie zostali unmutowani przed wyłączeniem bota. Tylko podczas startu, client nie jest jeszcze połączony i nie są ściągnięte dane, więc musi poczekać, aż wszystko będzie ściągnięte i wtedy dopiero można pobrać listę serwerów oraz userów. Przed tym nie są po prostu dostępni